planning/scenario_planning/parking/astar_search/CMakeLists.txt

```diff
@@ -1,48 +1,18 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(astar_search)
 
-add_compile_options(-std=c++14)
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
-find_package(catkin REQUIRED COMPONENTS
-  geometry_msgs
-  nav_msgs
-  std_msgs
-  tf2
-  tf2_geometry_msgs
-)
-
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES astar_search
-)
-
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
-)
-
-add_library(astar_search
-  src/astar_search.cpp
-)
-
-target_link_libraries(astar_search
-  ${catkin_LIBRARIES}
-)
-
-add_dependencies(astar_search
-  ${catkin_EXPORTED_TARGETS}
-)
+if(NOT CMAKE_CXX_STANDARD)
```

Comment:
[Minor]: Not sure if order particularly matters in this case but it seems like most poeple are specifying the compile options before the rest of the build process. Not sure if that should be strictly followed - everything compile so maybe its not an issue.

Reply:
This should be the top of this file. Thanks!